.github/ISSUE_TEMPLATE/user-assistance-template.md

```diff
@@ -0,0 +1,23 @@
+---
+name: eCR Viewer User Assistance 🙋
+about: Tracking ticket for debugging or troubleshooting efforts with STLT partners
+title: ""
+labels: Streamline eCR, "user assistance"
+assignees: ""
+---
+
+## User
+
+_Who is this request for?_
```

Comment:
a meta question: should this be a zenhub ticket instead of a github ticket? until it turns into a bug/task/feature, it feels a little off to have it here (weakly held opinion, but wanted to get your take)

Reply:
Zenhub ticket templates are pulled from github repo ticket templates, so it goes through github to get the template into zenhub for zenhub use.

Reply:
I don't see the templates available when creating a zenhub ticket - what am I doing wrong?

Reply:
@mcmcgrath13 do you get anything in the templates dropdown on Zenhub? If not, we may need to fiddle with your permissions - sometimes you're a member of the repo but not of Zenhub, and the visibility gets funky.

![Screenshot 2025-03-19 at 11 40 18 AM](https://github.com/user-attachments/assets/bf378d9c-8c34-44c7-8faa-bba4751a674e)

Reply:
That's for creating a new github issue. I was thinking these would be "zenhub issues", not github. Less risk of accidentally leaking user data/details